src/main/java/com/google/ehub/data/EntertainmentItemDatastore.java

```diff
@@ -104,51 +117,62 @@ public void addItemToDatastore(EntertainmentItem item) {
    * Queries all entertainment items found in Datastore.
    *
    * @param fetchOptions the fetch options used by the resulting query
-   * @return list with all entertainment items found in Datastore, the list will be empty if no
-   *     items were found
+   * @return list with all entertainment items found in Datastore, the list will
+   *     be empty if no items were found
    */
   public EntertainmentItemList queryAllItems(FetchOptions fetchOptions) {
-    return createItemListFromQuery(fetchOptions, new Query(ENTERTAINMENT_ITEM_KIND));
+    return createItemListFromQuery(fetchOptions,
+                                   new Query(ENTERTAINMENT_ITEM_KIND));
   }
 
   /**
-   * Queries all entertainment items found in Datastore and orders them based on title with the
-   * given sorting direction.
+   * Queries entertainment items with the specified title prefix and sorting
+   * direction.
    *
    * @param fetchOptions the fetch options used by the resulting query
-   * @param sortDirection the sort direction used to order the entertainment items based on title
-   * @return list with all the entertainment items following the specified sort direction, the list
-   *     will be empty if no items were found
+   * @param titlePrefix the title prefix used to filter the query
+   * @param sortDirection the sort direction used to order the entertainment
+   *     items based on title
+   * @return list with the entertainment items that match the title prefix and
+   *     sorting direction, the list will be empty if no items were found
    */
-  public EntertainmentItemList queryAllItemsWithTitleOrder(
-      FetchOptions fetchOptions, SortDirection sortDirection) {
-    return createItemListFromQuery(fetchOptions,
-        new Query(ENTERTAINMENT_ITEM_KIND).addSort(NORMALIZED_TITLE_PROPERTY_KEY, sortDirection));
+  public EntertainmentItemList
+  queryItemsByTitlePrefix(FetchOptions fetchOptions, String titlePrefix,
```

Comment:
This spacing looks off - is this from the autoformatter?

Also line 162-163 and 171-172

Reply:
Yeah I tried to see if there is a command to format a single file and found ```clang-format -i``` but it seems that adds some weird formatting styles. Do you know the command to format a single file without this issue?